app/Commands/Session/EndCommand.php

```
@@ -0,0 +1,174 @@
+        $lines = [];
+        $lines[] = "Session ended: " . now()->format('Y-m-d H:i:s');
+        $lines[] = "Project: {$project}";
+
+        if ($branch !== null) {
+            $lines[] = "Branch: {$branch}";
+        }
+
+        // Get commit count from session (approximate - last hour)
+        $commits = shell_exec('git log --since="1 hour ago" --oneline 2>/dev/null');
+        if ($commits !== null) {
+            $commitCount = count(array_filter(explode("\n", trim($commits))));
```

PHPStan (2.1.33)

Parameter #1 $string of function trim expects string, string|false given.


(argument.type)